portfolio/src/main/webapp/photography.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Photography</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <div id="content">
+      <h1>Photography</h1>
+      <h2>Why I love taking pictures</h2>
+      
+      <div class="image-thumbnails">
+        <a href="images/stanley-1.jpg"><img src="images/stanley-1.jpg"/></a>
+        <a href="images/stanley-2.jpg"><img src="images/stanley-2.jpg"/></a>
+        <a href="images/stanley-3.jpg"><img src="images/stanley-3.jpg"/></a>
+        <a href="images/stanley-4.jpg"><img src="images/stanley-4.jpg"/></a>
+        <a href="images/stanley-5.jpg"><img src="images/stanley-5.jpg"/></a>
+        <a href="images/stanley-6.jpg"><img src="images/stanley-6.jpg"/></a>
+        <a href="images/stanley-7.jpg"><img src="images/stanley-7.jpg"/></a>
+        <a href="images/stanley-8.jpg"><img src="images/stanley-8.jpg"/></a>
+        <a href="images/stanley-9.jpg"><img src="images/stanley-9.jpg"/></a>
+        <a href="images/stanley-10.jpg"><img src="images/stanley-10.jpg"/></a>
+        <a href="images/stanley-11.jpg"><img src="images/stanley-11.jpg"/></a>
+        <a href="images/stanley-12.jpg"><img src="images/stanley-12.jpg"/></a>
+        <a href="images/stanley-13.jpg"><img src="images/stanley-13.jpg"/></a>
```

Comment:
These images aren't present in this pull request, which is causing a broken page - http://screen/5cjmmZGJKCa. Maybe make these changes in an independent pull request?

Reply:
Oh, I see. :) this comment is on an older diff!

Looks good, thank you!

Reply:
this looks resolved by the file deletion :)